src/VisualStudio/Core/Def/Implementation/Experimentation/AnalyzerVsixSuggestedActionCallback.cs

```diff
@@ -69,7 +72,7 @@ public void OnSuggestedActionExecuted(SuggestedAction action)
                 !IsVsixInstalled() &&
                 IsCandidate())
             {
-                ShowInfoBarIfNecessary();
+                ThreadingContext.JoinableTaskFactory.Run(() => ShowInfoBarIfNecessaryAsync(CancellationToken.None));
```

Comment:
can we (manually) test to make sure this works safely and doesn't like deadlock or anything :D